x-pack/plugins/actions/server/index.ts

```diff
@@ -54,10 +55,61 @@ export { ACTION_SAVED_OBJECT_TYPE } from './constants/saved_objects';
 
 export const plugin = (initContext: PluginInitializerContext) => new ActionsPlugin(initContext);
 
+// Use a custom copy function here so we can perserve the telemetry provided for the deprecated config
+// See https://github.com/elastic/kibana/issues/112585#issuecomment-923715363
+function renameFromRootAndTrackDeprecatedUsage(
```

Comment:
I dont believe this is needed. please let me know if this comment makes sense: https://github.com/elastic/kibana/issues/112585#issuecomment-925804009

Reply:
Thanks for this clarification!